### PR TITLE
EVA-395:FAQ section images updated

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

**Comment by kandaj** [comment]
changes are made.

File: src/index.html [hunk 1]
```diff
@@ -476,6 +476,8 @@ <h3 class="panel-title study-browser-panel">
                         </div>
                         <div id="collapseFour" class="panel-collapse collapse">
                             <div class="panel-body">
+                                <img src="img/EVA_FIle_Level_Processing.svg" alt="EVA File Level Processing: Submitter brings VCF File(s) and metadata template to EVA -> Submitted data are validated against VCF specification and reference genome sequence ->
+                                 Acessions are sent to submitter,for referencing in publication for example -> EVA provide links to download the submitted data via the Study Browser"><br /><br />
```
Comment (cyenyxe) [inline src/index.html]: "Accessions" with double C.

File: src/index.html [hunk 2]
```diff
@@ -491,6 +493,8 @@ <h3 class="panel-title study-browser-panel">
                         <div id="collapseFive" class="panel-collapse collapse">
                             <div class="panel-body">
                                 <div>
+                                    <img src="img/EVA_Variant_Level_Normalisation.svg" alt="EVA Variant Level Processing: Submitted data from the EVA Study Browser -> Variants are merged,normalized and annotated for functional consequences and statistical values
```
Comment (cyenyxe) [inline src/index.html]: Comma between "merged" and "normalized"

File: src/index.html [hunk 3]
```diff
@@ -574,7 +578,7 @@ <h3 class="panel-title study-browser-panel">
                         </div>
                         <div id="collapseEight" class="panel-collapse collapse">
                             <div class="panel-body">
-                                <div><img src="img/EVA_Related_Projects.png"><br/><br/><br/></div>
+                                <div><img src="img/EVA_Related_Projects.png" alt="Collaborators:GEUVADIS European Exome Variant Server and Príncipe Felipe Research Cent​er​. Past Collaborators: The Genomics and Bioinformatics Platform of Andalusia​"><br/><br/><br/></div>
```
Comment (cyenyxe) [inline src/index.html]: Is "past" or "former" the correct word in English?

Comment (kandaj) [inline src/index.html]: both can be used i guess.  does former mean the one before the current one?.

File: src/index.html (hunk 2 above)
Comment (kandaj) [inline src/index.html]: fixed

File: src/index.html (hunk 3 above)
Comment (cyenyxe) [inline src/index.html]: Yes.

**Pull request merged**